src/openforms/api/drf_spectacular/hooks.py

```diff
@@ -65,3 +67,36 @@ def add_middleware_headers(result, generator, request, public):
     )
 
     return result
+
+
+CSRF_TOKEN_PARAMETER = build_parameter_type(
+    name=CSRF_TOKEN_HEADER_NAME,
+    schema={"type": "string"},
+    location=OpenApiParameter.HEADER,
+    required=True,
```

Comment:
I think it should only be required if session auth is the only authentication method for the endpoint. If token auth is an option, then the parameter is not required :grimacing: